llm4pddl/approaches/llm_open_loop_approach.py

```diff
@@ -182,3 +200,61 @@ def _llm_response_to_plan(response: LLMResponse, task: Task) -> Plan:
             action = f"({op} {objects_str})"
             plan.append(action)
         return plan
+
+    def embed_task(self, task: Task,
+                   embedding_model: SentenceTransformer) -> Embedding:
+        """Embeds a task using embedding_model.
+
+        Returns a numpy array.
+        """
+        # note: task_string includes the Q: and A: parts, not just task string
+        task_string = self._create_prompt(task)
+        # this line is a work around for now:
```

Comment:
this is a little risky if we change the Q and A stuff and forget to change things here. what would go wrong exactly if we just leave the task string alone, Q and A included? the only thing that really matters is that we're consistent about how we embed the training tasks and the eval tasks, right?